python_modules/dagster/dagster/_cli/project.py

```diff
@@ -182,6 +183,11 @@ def scaffold_code_location_command(context, name: str):
     default=False,
     help="Controls whether the project name can conflict with an existing PyPI package.",
 )
+@deprecated(
+    breaking_version="1.13",
+    additional_warn_text="Use 'dg scaffold' instead.",
+    emit_runtime_warning=True,
+)
```

Comment:
I think the actual replacement here is along the lines of `create-dagster project` right?